Update to reflect current software (#29257)
home-assistant/home-assistant.github.io: source/voice_control/thirteen-usd-voice-remote.markdown

world's most private voice assistant. Pick up the tiny device to talk to
your smart home. Issue commands and get responses!

## Required material

- Home Assistant 2023.10
- [Home Assistant Cloud](https://www.nabucasa.com) or a manually configured [Assist Pipeline](/voice_control/voice_remote_local_assistant)
- The password to your 2.4&nbsp;GHz Wi-Fi network
- Chrome (or a Chromium-based browser like Edge) on desktop (not Android/iOS) 
- [M5Stack ATOM Echo Development Kit](https://shop.m5stack.com/products/atom-echo-smart-speaker-dev-kit?ref=NabuCasa)
- USB-C cable to connect the ATOM Echo

## Installing the openWakeWord add-on

As a first step, you need to install the openWakeWord add on. This must be installed before setting up the ATOM Echo.

1. Go to {% my supervisor_addon addon="openwakeword" title="**Settings** > **Add-ons** > **openWakeWord**" %} and select **Install**.
2. Start the add-on.
3. Go to {% my integrations title="**Settings** > **Devices & Services**" %}.
   - Under **Discovered**, you should now see the **openWakeWord** integration.
   - Select **Configure** and **Submit**.

> 4. To define the wake word enging, under **Wake word**, select **openwakeword**. 

Reviewer: enging -> engine

> 4. To define the wake word enging, under **Wake word**, select **openwakeword**. 

Reviewer: Should we mention this is in the Voice Assistants settings within a pipeline?

Author: oh, yes. something went missing here. Thank you for the catch.

> 4. To define the wake word enging, under **Wake word**, select **openwakeword**. 

Author: ```suggestion
4. To define the wake word engine, under **Wake word**, select **openwakeword**. 
```

   - Then, select **ok, nabu**.

>    - To confirm your choices, select **Create**.

Reviewer: Or Update if the pipeline already existed.

Author: True. The reason I did not mention it is because in this procedure, people are instructed to create a new one. Because this should work for both the newbies and the more experienced users.

>    - To confirm your choices, select **Create**.

Author: ```suggestion
   - If you created a new assistant, select **Create**.
   - If you edited an existing assistant, select **Update**.
```

- USB-C cable to connect the ATOM Echo

## Installing the openWakeWord add-on

As a first step, you need to install the openWakeWord add on. This must be installed before setting up the ATOM Echo.

1. Go to {% my supervisor_addon addon="openwakeword" title="**Settings** > **Add-ons** > **openWakeWord**" %} and select **Install**.
2. Start the add-on.
3. Go to {% my integrations title="**Settings** > **Devices & Services**" %}.
   - Under **Discovered**, you should now see the **openWakeWord** integration.
   - Select **Configure** and **Submit**.
   - **Result**: You have successfully installed the openWakeWord add-on and integration.

## Adding a wake word to your voice assistant

1. Go to {% my voice_assistants title="**Settings** > **Voice assistants**" %} and select **Add assistant**.
2. Give your assistant a name, for example the wake word you are going to use.
3. Select the language you are going to use to speak to Home Assistant.
4. Under **Text-to-speech**, select the language and voice you want Home Assistant to use when speaking to you.
5. To define the wake word engine, under **Wake word**, select **openwakeword**. 

>    - Then, select **ok, nabu**.

Reviewer: "ok nabu" (no comma)

>    - Then, select **ok, nabu**.

Author: ```suggestion
   - Then, select **ok nabu**.
```

   - Under **M5Stack ATOM Echo**, select **1 device**.
   ![Open My link](/images/assist/esp32-atom_silence_detection_01.png)

## Disabling wake word and use push-to-talk

> 1. If you do not want to use a wake work, but prefer to use the microphone by pressing a button, you can disable the wake word.

Reviewer: work -> word

> 1. If you do not want to use a wake work, but prefer to use the microphone by pressing a button, you can disable the wake word.

Author: ```suggestion
1. If you do not want to use a wake word, but prefer to use the microphone by pressing a button, you can disable the wake word.
```

> 1. If you do not want to use a wake work, but prefer to use the microphone by pressing a button, you can disable the wake word.

Author: ```suggestion
1. If you do not want to use a wake word, but prefer to use the microphone by pressing a button, you can disable the wake word.
```

2. Go to {% my integrations title="**Settings** > **Devices & Services**" %} and select the **ESPHome** integration.
   - Under **M5Stack ATOM Echo**, select **1 device**.
3. Under **Controls**, disable **Use wake word**.

>    ![Open My link](/images/assist/wake_word_disable_on_atom_echo.png)

Reviewer: "Open My Link" should be description of image

>    ![Open My link](/images/assist/wake_word_disable_on_atom_echo.png)

Author: ```suggestion
   ![Toggle to enable/disable wake word](/images/assist/wake_word_disable_on_atom_echo.png)
```

4. To start using push-to-talk, press the flat button with rounded shape on your ATOM Echo.
   - The rectangular button on the side is the reset button. Do not press that one.
   - As soon as you press the button, the LED will start blinking in blue. If it does not light up, press again.
   - While you are speaking, the blue LED is pulsing.
   - Once the intent has been processed, the LED lights up in green and Home Assistant confirms the action.

## Troubleshooting

Are things not working as expected?

- Checkout the [general troubleshooting section for Assist](/voice_control/troubleshooting/).
- You think there is a problem with noise or volume? Checkout the procedure below.

### Tweaking the ATOM Echo configuration

1. Make sure you have [access to your configuration files](/common-tasks/os/#configuring-access-to-files).

> 2. Eidt the general configuration:

Reviewer: Eidt -> Edit

> 2. Eidt the general configuration:

Author: ```suggestion
2. Edit the general configuration:
```
